Core/include/Acts/EventData/Measurement.hpp

```diff
@@ -80,11 +80,11 @@ class Measurement {
 
  public:
   /// type of the vector containing the parameter values
-  using ParameterVector = typename ParamSet::ParameterVector;
+  using ParameterVector = typename ParamSet::ParametersVector;
```

Comment:
```suggestion
  using ParametersVector = typename ParamSet::ParametersVector;
```

Reply:
I would prefer to leave this as-is. Otherwise, I also have to change all `Measurement` uses. Should we open another PR as a follow-up for consistency?

Reply:
No need to add it to this PR, but a follow-up would be nice.

Reply:
@FabianKlimpel Turns out this only adds ~20 lines of changes. I updated the local typedefs in `Measurement` as part of this PR.